acetone/utils.py

```diff
@@ -0,0 +1,17 @@
+import importlib
+
```

Comment:
Please don't name the file utils :) resolver or anything else would do just fine.